linux/ibus-keyman/tests/run-tests.sh

```diff
@@ -106,5 +118,5 @@ echo "kill -9 $!" >> $PID_FILE
 sleep 1s
 
 echo "Starting tests..."
-./ibus-keyman-tests ${ARG_K-} ${ARG_TAP-} --directory $TESTDIR "${TESTFILES[@]}"
+./ibus-keyman-tests ${ARG_K-} ${ARG_TAP-} ${ARG_SURROUNDING_TEXT-} ${ARG_NO_SURROUNDING_TEXT-} --directory $TESTDIR "${TESTFILES[@]}"
```

Comment:
You pass -k and --tap in here but I can't see how those are used in testfixture.cpp?

Reply:
They are consumed by the GLib testing framework.